lib/rubocop/cop/sorbet/one_ancestor_per_line.rb

```diff
@@ -0,0 +1,81 @@
+# encoding: utf-8
+# frozen_string_literal: true
+
+require 'rubocop'
+
+module RuboCop
+  module Cop
+    module Sorbet
+      # This cop ensures one ancestor per requires_ancestor line
+      # rather than chaining them as a comma-separated list.
+      #
+      # @example
+      #
+      #   # bad
+      #   module SomeModule
+      #     requires_ancestor Kernel, Minitest::Assertions
+      #   end
+      #
+      #   # good
+      #   module SomeModule
+      #     requires_ancestor Kernel
+      #     requires_ancestor Minitest::Assertions
+      #   end
+      class OneAncestorPerLine < RuboCop::Cop::Cop
+        MSG = 'Cannot require more than one ancestor per line'
+
+        def_node_search :requires_ancestors, <<~PATTERN
+          (send nil? :requires_ancestor ...)
+        PATTERN
+
+        def_node_matcher :more_than_one_ancestor, <<~PATTERN
+          (send nil? :requires_ancestor const const+)
+        PATTERN
+
+        def_node_search :abstract?, <<~PATTERN
+          (send nil? :abstract!)
+        PATTERN
+
+        def on_module(node)
+          return unless node.body
+          return unless requires_ancestors(node)
+          process_node(node)
+        end
+
+        def on_class(node)
+          return unless abstract?(node)
+          return unless requires_ancestors(node)
+          process_node(node)
+        end
+
+        def autocorrect(node)
+          -> (corrector) do
+            ra_call = node.parent
+            split_ra_calls = ra_call.source.gsub(/,\s+/, new_ra_line(ra_call))
+            corrector.replace(ra_call, split_ra_calls)
+          end
+        end
+
+        private
+
+        def process_node(node)
+          requires_ancestors(node).each do |ra|
+            add_offense(ra.child_nodes[1]) if more_than_one_ancestor(ra)
+          end
+        end
+
+        def indents(count)
+          " " * count
+        end
+
+        def new_ra_line(ra_call)
+          out = StringIO.new
+          out << "\n"
+          out << indents(ra_call.loc.column) # Match indent
+          out << "requires_ancestor "
+          out.string
```

Comment:
NIT: this is a good opportunity to use `rjust(count, char)` (right justify). It justifies from the right using `count` number of `char`s. The default is blank spaces.

We can probably skip using `StringIO` too and use an unfrozen string.
```rb
out = +"\n" # create unfrozen string
out << "requires_ancestor ".rjust(ra_call.loc.column)
out
```

Reply:
Neat! TIL
Yeah I 'borrowed' the StringIO bit from another cop to make the `indents(...)` call a little 'cleaner' looking. `rjust` should help me clean this up! Thanks!

Reply:
Interesting! Rjust doesn't do what you'd expect.

```
rjust(integer, padstr=' ') → new_str

If integer is greater than the length of str, returns a new String of length integer with str right justified and padded with padstr; otherwise, returns str.

"hello".rjust(4) #=> "hello"
```

I will tidy this up a bit though.